createNativeStackNavigator.js

```diff
@@ -27,18 +27,36 @@ function renderComponentOrThunk(componentOrThunk, props) {
 
 class StackView extends React.Component {
   _removeScene = route => {
+    console.warn('REMOVE SCENE');
     this.props.navigation.dispatch(StackActions.pop({ key: route.key }));
   };
 
-  _onSceneFocus = (route, descriptor) => {
+  _onAppear = (route, descriptor) => {
     descriptor.options &&
       descriptor.options.onAppear &&
       descriptor.options.onAppear();
     this.props.navigation.dispatch(
-      StackActions.completeTransition({ toChildKey: route.key })
+      StackActions.completeTransition({
+        toChildKey: route.key,
+        key: this.props.navigation.state.key,
+      })
     );
   };
 
+  _onFinishTransitioning = () => {
+    console.warn('FINISH TRANSITIONING');
```

Comment:
@kmagiera looks like this was left here accidentally, it's showing on my logs since I updated earlier today